src/build.py

```diff
@@ -625,19 +626,36 @@ def Summary(repos):
     UploadToCloud('lkgr', 'git/lkgr', 'lkgr')
 
 
+def ParseArgs():
+  parser = argparse.ArgumentParser(
+      description='Wasm waterfall top-level CI script')
+  parser.add_argument('--no-sync', dest='sync',
+                      default=True, action='store_false',
+                      help='Skip fetching and checking out source repos')
+  parser.add_argument(
+      '--no-build', dest='build', default=True, action='store_false',
+      help='Skip building source repos (also skips V8 and LLVM unit tests)')
+  return parser.parse_args()
+
+
 def main():
+  options = ParseArgs()
```

Comment:
Could you instead follow the approach other `.py` files do in this repo: call `ParseArgs` only when not `__main__`? Right now it works because you have all-defaults, but won't work otherwise.